needles/chrome_example_com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "tags": [
+    "chrome_example_com"

```

Comment:
The PNG for this one will need updating when we fix the theme to not make inactive tabs have black-on-black.

Reply:
Yup, that’s noted and can be done as a follow-up.